### PR TITLE
Add release notes for 17.8.0

### PR DESCRIPTION
**Pull request by jrha**

### CODE REVIEW

File: _posts/2017-10-30-quattor-17.8.0.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: Quattor 17.8.0 released
+category: news
+author: James Adams
+---
+
+Packages are available from our [yum repository](http://yum.quattor.org/17.8.0/), both the RPMs and the repository metadata are signed with [my GPG key](http://yum.quattor.org/GPG/RPM-GPG-KEY-quattor-jrha).
+
+As always, many thanks to everyone who contributed! We merged 30 pull requests and resolved 22 issues.
+
+The most notable and important change in this release is the overhaul of `ncm-network` which may require sites to change some configuration values.M
```
Comment (stdweird) [inline _posts/2017-10-30-quattor-17.8.0.md]: line too long?
the line also ends with a `M`

Comment (jrha) [reply _posts/2017-10-30-quattor-17.8.0.md]: Blargh, removed the `M`.
There is no line length limit with markdown, line wrapping is expected.

Comment (stdweird) [reply _posts/2017-10-30-quattor-17.8.0.md]: but there is in github and makes changes complicated to review. (btw, there's also no limit in pan or perl 😉

**Comment by jrha** [comment]
I'm going to merge this so I can announce the release.

**Pull request merged**